apps/web/src/pages/EmploymentEquityPage/EquityOptions.tsx

```diff
@@ -101,12 +121,12 @@ const EquityOptions: React.FC<EquityOptionsProps> = ({
                   )}
                 />
               )}
-              {resolvedIndigenous && (
-                <EquityOption
+              {resolvedIndigenousCommunities.length && (
```

Comment:
When checking array lengths in react, we need to use a full ternary or else the length will get output to the screen.

```tsx
{resolvedIndigenousCommunities.length ? (
 ...
) : null}
```

Reply:
Oops!  Good catch.   Fixed in [fixes from Eric's review](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5461/commits/e7a019aa742654d22f4aa3f02ab5dd1297ae656c).